compiler-collection.js

```diff
@@ -0,0 +1,47 @@
+module.exports = (expireAfterSeconds) => {
+  const expireMs = expireAfterSeconds * 1000;
+  const compilers = new Map();
+
+  if (expireAfterSeconds) {
+    setInterval(removeExpiredCompilers, 30 * 1000);
```

Comment:
You could probably make the check interval a parameter? Or at least a const.

Reply:
It doesn't significantly affect anything i.e. we probably wouldn't notice the difference between 1 second and 30 minutes. I don't think it's worth making configurable. Naming it in a const isn't a bad idea, but I think the code makes it obvious what this is for and a name like "expiredCompilersCheckIntervalInSeconds" doesn't really add much.